fairlearn/postprocessing/_interpolated_thresholder.py

```diff
@@ -57,6 +89,18 @@ def fit(self, X, y, **kwargs):
         if self.estimator is None:
             raise ValueError(BASE_ESTIMATOR_NONE_ERROR_MESSAGE)
 
+        if self.predict_method == "deprecated":
+            warn(
+                "'predict_method' is changed from 'predict' to 'auto'. "
+                "Explicitly pass `predict_method='predict' to replicate the "
+                "old behavior, or pass `predict_method='auto' to silence "
```

Comment:
maybe:
..., or pass another valid value to silence
(the point being that 'predict_proba' would also silence the behavior)